lib/yi_appium_caps_util.rb

```diff
@@ -181,11 +187,23 @@ def update_ios_caps (output_data)
         elsif (xcodeBuildVersion.to_f>8) then
             puts "Change to xcode version to xcode 7! Current version is: "+xcodeBuildVersion
         end
+      end  
+    rescue Exception => ex
+      puts "An error of type #{ex.class} happened, message is #{ex.message}"
+      raise
     end
 
+    def update_mac_caps (output_data)
+      #Replace value of deviceName
+      output_data['caps']['deviceName'] = "macOS device"
+      #Replace value of youiEngineAppAddress
+      output_data['caps']['youiEngineAppAddress'] = "localhost"
+    rescue RuntimeError => ex
```

Comment:
I'm not sure why you have to rescue `RuntimeError` and `Exception` errors separately 🤔 ? When will you get one over there other?

Reply:
Also, you can rescue multiple exceptions in the same line
```
rescure RuntimeError, Exception =>ex
    raise 'An error of type #{ex.class} happened, message is #{ex.message}'
```

Reply:
I'm not sure, this came from Mohamed, I just copied that block of code to add a new platform.
Sure I can do that.